### DISCUSSION TITLE
Disable dashboard and listeners service

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx-operator#614] [General]
Is there anyway to Disable dashboard and listeners services created by emqx crd? Do not want to create these service automatically

Thanks..

**Comment by Rory-Z** [comment]
Could you please tell me your EMQX‘s version and EMQX Operator’s version ？

**Reply by alexa22013** [reply]
emqx version 5.0.14, operator 2.1.0

**Reply by Rory-Z** [reply]
You can disable EMQX listener, just add `listeners.tcp.default.enable = false` in `.spec.bootstrapConfig`, but don't disable EMQX Dashboard, because this will disable EMQX Dashboard listener port, and EMQX API will not work, and readiness probe will failed

**Reply by alexa22013** [reply]
Sorry but I want to disable the services created by listenerServiceTemplate and dashboardTemplate

**Reply by Rory-Z** [reply]
In general, we do not recommend that users create their own service, as the EMQX Operator will update the service according to the EMQX configuration, like when user change the listener ports via EMQX Dashboard, EMQX Operator will update service port.

Could you please tell me why you want disable the `listenerServiceTemplate` and `dashboardTemplate` ?  If it makes sense, we will consider implementing it.

**Comment by zhongwencool** [comment]
I'm a bit curious why you would want to disable the dashboard, do you need to disable the API too?

**Reply by alexa22013** [reply]
basically I want to disable the services created by listenerServiceTemplate and dashboardTemplate, so I can define my own service
